iceberg/iceberg-handler/src/test/queries/negative/create_iceberg_table_stored_as_with_serdeproperties_failure.q

```diff
@@ -0,0 +1,3 @@
+set hive.vectorized.execution.enabled=false;
```

Comment:
Can we add a negative test somewhere to verify that it fails for other storage handler types? e.g. stored by kudu stored as orc. Not sure what the best place would be to put it

Reply:
I've added one to the HBase qtests.